service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/Query.java

```diff
@@ -334,13 +336,98 @@ public QueryResponse sendQueryRequest(IotHubConnectionString iotHubConnectionStr
         return this.queryResponse;
     }
 
+    /**
+     * Sends request for the query to the IotHub.
+     *
+     * @param tokenCredential The custom {@link TokenCredential} that will provide authentication tokens to
+     *                                    this library when they are needed.
+     * @param tokenCredentialType The type of authentication tokens that the provided {@link TokenCredential}
+     *                          implementation will always give.
+     * @param url URL to Query on.
+     * @param method HTTP Method for the requesting a query.
+     * @param httpConnectTimeout the http connect timeout to use for this request.
+     * @param httpReadTimeout the http read timeout to use for this request.
+     * @param proxy the proxy to use, or null if no proxy should be used.
+     * @return QueryResponse object which holds the response Iterator.
+     * @throws IOException If any of the input parameters are not valid.
+     * @throws IotHubException If HTTP response other then status ok is received.
+     */
+    public QueryResponse sendQueryRequest(TokenCredential tokenCredential,
```

Comment:
Elsewhere we seem to put the first parameter on the next line, tabbed in just once.